### DISCUSSION TITLE
Windows ARM64 support

### DISCUSSION THREAD

**Discussion by gratianlup** [ente-io/ente#3404] [Enhancements]
Tried to install the app on one of those new ARM64 Windows laptops and the setup reported it's not a supported architecture.
If not a native ARM64 build, could at least the setup be changed so it allows the install and the app would run through emulation then (most likely it should be fine).

Thanks!

**Comment by ua741** [comment]
Related issue: https://github.com/flutter/flutter/issues/62597

**Comment by gratianlup** [comment]
An update I tried the "portable" release that doesn't use a setup, and the app runs fine through the emulation.
